src/solvers/dg_unstructured_quad/dg_2d.jl

```diff
@@ -0,0 +1,341 @@
+# This method is called when a SemidiscretizationHyperbolic is constructed.
+# It constructs the basic `cache` used throughout the simulation to compute
+# the RHS etc.
+function create_cache(mesh::UnstructuredQuadMesh, equations::AbstractEquations,
+                      dg::DG, RealT, uEltype=RealT)
+
+  poly_deg = nnodes(dg.basis) - 1
```

Comment:
```suggestion
  poly_deg = polydeg(dg)
```
Also, I'd prefer not to use `poly_deg`, but rather `polydeg` or `polydeg_` to be consistent